test/format/config.c

```diff
@@ -358,6 +375,25 @@ config_run(void)
 {
     config_random(tables[0], false); /* Configure the remaining global name space. */
 
+    /*
+     * Limit the number of tables to 5 if realloc exact and realloc malloc are both on and not all
+     * explicitly set to reduce the running time to acceptable level.
+     */
+    if ((!config_explicit(NULL, "debug.realloc_exact") ||
+          !config_explicit(NULL, "debug.realloc_malloc")) &&
+      GV(DEBUG_REALLOC_EXACT) && GV(DEBUG_REALLOC_MALLOC) && ntables > 5) {
+        ntables = 5;
+        /*
+         * This has not effect just to overwrite the config in memory so that we can dump the
+         * correct config.
+         */
+        config_single(NULL, "runs.tables=5", true);
```

Comment:
To use the suggested macro you'll need to sprintf into a buf.

Reply:
I used another approach with STR(x) macro